src/lib/Sympa/Config.pm

```diff
@@ -965,7 +963,9 @@ sub _merge_changes_paragraph {
 
 sub get_id {
     my $that = shift->{context};
-    (ref $that eq 'Sympa::List') ? $that->get_id : (defined $that) ? $that : '';
+          (ref $that eq 'Sympa::List') ? $that->get_id
```

Comment:
Well, IMHO, that's not the weirdest indentation that Perl::Tidy asked for. I'm not a big fan of src/lib/Sympa/Database.pm L.77 for ex.

Also, I can't find it anymore (too much changed files), but I remember there's an `if` that have its closing `) {` not on the same level of indentation and that was weird since Perl::Tidy made me change a lot of closing `) {` indentation.

Reply:
I see. How weird perltidy does, it does its best.